### DISCUSSION TITLE
Storing configurations

### DISCUSSION THREAD

**Discussion by verasbr** [ATVWGS/tenant-management-framework#12] [Q&A] [answered]
https://github.com/ATVWGS/tenant-management-framework#245-storing-configurations

Wondering what is the best way to store configurations for different tenants, a single repo with a Directory for each tenant and the config in there? Can you still leverage Prerequisite setting if you have configs in Directories. Ie we want to have default config dir for all the tenants to use and then one for Prod and one for Non-Prod.

**Comment by ATVWGS** [comment, marked as answer]
Sorry for the late response :) 
We have implemented it like you want to. We are using one repo containing different config folders (default, dev, qa, prod, etc.) and have prerequisites set between these configurations. E.g. dev-configuration needs also default-configuration to be loaded.
